source/extensions/filters/http/grpc_http1_reverse_bridge_transcoder/filter.cc

```diff
@@ -27,10 +29,12 @@ const auto GrpcToJsonFailed = "Failed to transcode gRPC to JSON.";
 const auto JsonToGrpcFailed = "Failed to transcode JSON to gRPC.";
 } // namespace Errors
 
+/*
 Http::RegisterCustomInlineHeader<Http::CustomInlineHeaderRegistry::Type::RequestHeaders>
```

Comment:
Remove if it is not used.

Reply:
I commented it out because I am not entirely sure what it does. Do you have any Idea what it is supposed todo?

Reply:
Then I'll remove it. It is commented out for quite a while.